src/main/kotlin/app/batch/HBaseWriter.kt

```diff
@@ -0,0 +1,22 @@
+package app.batch
```

Comment:
Do we need HBase writer in this project?

Reply:
Yes, it writes to hbase.